tests/rkt_run_pod_manifest_test.go

```diff
@@ -111,6 +90,106 @@ type imagePatch struct {
 	patches []string
 }
 
+type podManifestTest struct {
+	// [image name]:[image patches]
+	images         []imagePatch
+	podManifest    *schema.PodManifest
+	shouldSuccess  bool
+	expectedResult string
+	cgroup         string
+}
+
+func runSinglePodManifestTest(ctx *rktRunCtx, t *testing.T, tmpdir string, i int, tt podManifestTest) {
+	if tt.cgroup != "" && !cgroup.IsIsolatorSupported(tt.cgroup) {
+		t.Logf("Skip test #%v: cgroup %s not supported", i, tt.cgroup)
```

Comment:
Nah, that would skip entire TestPodManifest and we want to skip just one... err... I dunno what's the name... test in test case?